KW/settings.py

```diff
@@ -81,7 +81,7 @@
 CELERY_BEAT_SCHEDULE = {
     "all_user_srs_every_hour": {
         "task": "kw_webapp.srs.all_srs",
-        "schedule": crontab(minute="*/15"),
+        "schedule": crontab(minute="1"),
```

Comment:
Are you sure that it should be crontab(minute="1"), not crontab(minute="0")?
Now it will execute on 14:01, 15:01, etc.

Reply:
Yup this is intended :)  The reviews are scheduled for the top of the hour (minute 0), so kicking off this process at minute 1 will produce the intended effect. I could also set the reviews to minute 59 of the previous hour, then have this scheduled to run on minute 0, but this SRS process takes ~1.5 minutes to complete anyhow.